indigo/analysis/sentence_caser.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+import unicodedata
+from lxml import etree
+
+from indigo.plugins import LocaleBasedMatcher, plugins
+
+
+@plugins.register('sentence-caser')
+class BaseSentenceCaser(LocaleBasedMatcher):
+    """ Sentence cases headings in a document.
+    """
+    terms = None
+    normalized_terms = None
+
+    def sentence_case_headings_in_document(self, document):
+        accented_terms = document.language.accented_terms.first()
+        self.terms = accented_terms.terms if accented_terms else []
+        self.terms.sort(key=lambda x: len(x), reverse=True)
+        self.normalized_terms = [''.join(c for c in unicodedata.normalize('NFD', t) if unicodedata.category(c) != 'Mn').lower()
+                                 for t in self.terms]
+        root = etree.fromstring(document.content.encode('utf-8'))
+        for heading in etree.XPath('//a:heading', namespaces={'a': root.nsmap[None]})(root):
+            self.capitalized = False
+            for elem in heading.getiterator():
```

Comment:
the docs say `getiterator()` is deprecated and you should use `iter()` instead -- looks the same.

Reply:
You don't want to descend into footnote elements, so you should check elem.tag and ignore footnotes.

In your loom, note that your footnote goes from `FN` to `fn`

Reply:
all better 😁 

<img width="470" alt="image" src="https://github.com/user-attachments/assets/5e1fceba-75d7-4fa1-ac24-1b6f94065689">